omod/src/main/java/org/openmrs/module/spa/SpaActivator.java

```diff
@@ -11,19 +11,57 @@
 
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
+import org.openmrs.api.AdministrationService;
+import org.openmrs.api.context.Context;
 import org.openmrs.module.BaseModuleActivator;
+import org.openmrs.module.ModuleConstants;
+import org.openmrs.module.ModuleException;
+import org.openmrs.util.OpenmrsUtil;
+
+import java.io.File;
 
 public class SpaActivator extends BaseModuleActivator {
 
 	private final Log log = LogFactory.getLog(getClass());
```

Comment:
And then i would go for SPA_STATIC_FILES_DIR_DEFAULT_VALUE instead of FRONTENT_DIRECTORY

Reply:
We're trying to build the infrastructure for what will be the new frontend to OpenMRS.